target/linux/ath79/image/generic.mk

```diff
@@ -999,6 +999,21 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/meraki_mr16
+  SOC := ar7161
+  DEVICE_VENDOR := Meraki
+  DEVICE_MODEL := MR16
+  ROOTFS_SIZE := 13440k
+  IMAGE_SIZE := 15680k
+  DEVICE_PACKAGES := kmod-spi-gpio kmod-owl-loader
+  IMAGE/kernel.bin := append-kernel
+  IMAGE/rootfs.bin := append-rootfs | pad-rootfs
+  IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | pad-to $$$$(ROOTFS_SIZE) | append-kernel | append-metadata | check-size
```

Comment:
Please wrap so one line contains max. 80 characters. Hanging indent with one tab.

Reply:
Done in 8d699c6.